packages/suite-desktop/src-electron/modules/index.ts

```diff
@@ -47,6 +46,32 @@ type ModuleInit = (dependencies: Dependencies) => ModuleLoad | void;
 
 export type Module = ModuleInit;
 
+export const initTorModule = async (dependencies: Dependencies) => {
+    const { logger } = global;
+
+    logger.info('modules', `Initializing tor module`);
+
+    try {
+        const m = await require(`./modules/tor`);
+        const initModule: Module = m.default;
+        const torModule = initModule(dependencies);
```

Comment:
Given that Tor module is now handled differently than the regular modules, I think it could be simplified, e.g.
- not imported dynamically (because it must be present in both dev and prod anyway)
- not split into `init` and `load` phases as they are called right after each other anyway (and it's between `handshake` and `loadModules`)
- the `initModule` could always return non-null, as the `loaded` check could go away from `tor` module, as it could be called only once during the application run (not 100% sure about that yet)
- from the previous points, it almost seems that `tor.ts/load` could be directly bound in `app.ts` to <code>ipcMain.handle<b>Once</b>('handshake/load-tor', ...)</code>, without needing `tor.ts/init` or `modules/index.ts/initTorModule`. Does it make sense?

Reply:
mmmm :thinking: I am looking at it, I think it should be possible.

Reply:
Yes, it could be simplified. Less code less problems. What do you think about 5063c3dd8dce9c20344e4f7808f678ec6a6fbc35 ?